README.rst

```diff
@@ -400,6 +400,47 @@ You can redefine the on_validation_error hook in your PydanticView
             return json_response(data=errors, status=400)
 
 
+Custom Open API Schema
+----------------------
+
+You can define a hook to customize the schema that is generated for each
+function to clarify or adjust the computed schema. For this, ensure that the
+view's get/put/delete/etc. property is equipped with a `__modify_schema__`
+attribute, which is invoked as a callback each time the schema is generated.
+
+.. code-block:: python3
+    def add_custom_example(oas, oas_path, view, oas_operation):
```

Comment:
Actually, this brings up a bit of an unresolved interface problem. These 4 parameters are _somewhat_ arbitrary. Would it be better to provide a `namedtuple` so that additional context can be added later, if it turns out that it was missing?